internal/pkg/server/standAlone_dev.go

```diff
@@ -0,0 +1,138 @@
+// Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+// or more contributor license agreements. Licensed under the Elastic License;
+// you may not use this file except in compliance with the Elastic License.
+
+//go:build dev
+
+package server
+
+import (
+	"bytes"
+	"context"
+	"encoding/json"
+	"errors"
+	"fmt"
+	"net/http"
+	"os"
+	"time"
+
+	"github.com/elastic/fleet-server/v7/internal/pkg/api"
+	"github.com/elastic/fleet-server/v7/internal/pkg/bulk"
+	"github.com/elastic/fleet-server/v7/internal/pkg/dl"
+	"github.com/elastic/fleet-server/v7/internal/pkg/es"
+	"github.com/elastic/fleet-server/v7/internal/pkg/model"
+	"github.com/elastic/fleet-server/v7/internal/pkg/policy"
+	"github.com/elastic/fleet-server/v7/internal/pkg/sqn"
+	"github.com/rs/zerolog/log"
+)
+
+func (f *Fleet) standAloneSetup(ctx context.Context, bulker bulk.Bulk, sm policy.SelfMonitor, policyID, agentID string) (*model.Agent, error) {
+	err := sm.Run(ctx)
+	if err != nil {
+		return nil, fmt.Errorf("unable to run self monitor for stand alone setup: %w", err)
+	}
+	policy := sm.Policy()
+	log.Debug().Msgf("Policy Data: %s", string(policy.Data))
+	// TODO use policy from self monitor
```

Comment:
We should make a decision about the source of truth for the fleet-server in stand alone.
Do we want to :
1. feed the retrieved policy back into the config that get's executed
2. run only what's in the local `fleet-server.yml` config file

Option 1 will more closely mimic how the fleet-server runs under the agent. But may need more bootstrapping
Option 2 is much simpler, and we can easily fake any acks to policy_change or reassign actions.

Reply:
After giving this more thought, i'm leaning towards option 2 and just using config when the agent is in stand-alone mode.
The main purpose of option 1 would be to mimic running under the agent, but that scenario should be tested anyways and I don't think a stand-alone fleet-server gains much value if it needs to bootstrap in order to get remote config.
Also if we want to make a fleet service the config should not be exposed through kibana